src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line import/extensions
+import CalcContainer from './components/Calculator.js';
```

Comment:
This component is not connected properly. Kindly accept my suggestion.

```suggestion
import CalcContainer from './components/calculator.js';
```

Reply:
Thanks a lot, I'll do that 😄 .